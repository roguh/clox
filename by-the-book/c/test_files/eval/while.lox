while (false) {
    print("nope");
}
var i = 1;
var x = 1;
while (i < 18) {
    i = i + 1 ;
    x = x * 2;
    print(i);
    print(x);
    if (i > 20) { return; }
}
print("END OF LOOP:");
print(x);
