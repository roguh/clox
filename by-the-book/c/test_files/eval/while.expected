2
2
3
4
4
8
5
16
6
32
7
64
8
128
9
256
10
512
11
1024
12
2048
13
4096
14
8192
15
16384
16
32768
17
65536
18
131072
END OF LOOP:
131072
